### ISSUE TITLE
GLKView::deleteDrawable equivalent in MGLKView

### ISSUE THREAD

**Issue by mb12** [kakashidinho/metalangle#60]
There is no equivalent of the GLKView::deleteDrawable in MGLKView.
Is this no longer needed with Metal backend?

https://developer.apple.com/documentation/glkit/glkview/1615569-deletedrawable?language=objc

Currently our application calls when the user presses the home button on the iPhone(equivalent swipe up home gesture).

**Comment by kakashidinho** [comment]
Right. I haven’t implemented the equivalent function yet. Will do in future. For now, the framebuffer will be alive as long as your view is alive.
